simpleviewinc/keg-components
# Zen 255 run yarn format

**Ticket**: [ZEN-255](https://jira.simpleviewtools.com/browse/ZEN-255)

## Context

* Running `yarn format` and manually fixing any errors that was caught via eslint

## Goal

* make code look pretty

## Updates

* a bunch of files has been formatted
* i had some manual changes on some of the files because eslint doesn't fix some rules automatically
     * unused variable
     * 1 char var
     * missing imports

## Testing
* pull branch
* run `yarn sb` - verify the components still runs and behaves the same as before
* run `yarn test` -verify test passes
